Resources/docs/adapters/awss3.md

```diff
@@ -63,3 +110,52 @@ knp_gaufrette:
                 options:
                     directory: 'profile_photos'
 ```
+
+## Adapter for Amazon S3 SDK v1 (DEPRECATED by Amazon)
+
+```
+composer require amazonwebservices/aws-sdk-for-php
+```
+
+### Service definition
+
+To use the Amazon S3 adapter you need to provide a valid `AmazonS3` instance (as defined in the Amazon SDK). This can
+easily be set up as using Symfony's service configuration:
+
+``` yaml
+# app/config/config.yml
+services:
+    amazonS3:
+        class: AmazonS3
+        arguments:
+            options:
+                key: '%aws_key%'
+                secret: '%aws_secret_key%'
+```
+
+### Parameters
+
+ * `amazon_s3_id`: the id of the AmazonS3 service used for the underlying connection
+ * `bucket_name`: the name of the bucket to use
+ * `options`: additional (optional) settings
+   * `directory`: the directory to use, within the specified bucket
+   * `region`
+   * `create`
+
+### Example
+
+Once the service is set up use its key as the amazon_s3_id` in the gaufrette configuration:
```

Comment:
looks like there's a missing backquote on this line

Reply:
fixed, thx